src/components/global/Column.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { Droppable } from "react-beautiful-dnd";
+import "../../stylesheets/components/global/column.scss";
+import Task from "./Task/Task";
+
+function Column(props) {
+  const { column, tasks, subTasks, handleList } = props;
+
+  const [open, setOpened] = React.useState(false);
+
+  const onButtonClick = () => {
+    setOpened(!open);
+  };
+
+  return (
+    // This is where the droppable region should be
+    <div className="container">
```

Comment:
The CSS class name convention we are following is BEM. So this outer div should be called 'column', which is a kebab case version of the component file. Then, any containing divs would follow this format: 'column__classname'